ComparePaymentProcessors.md

```diff
@@ -0,0 +1,95 @@
+
+Fortunately, the Bitcoin Network does not use or collect these types of personal information, and therefore neither does BTCPay. 
+
+----
+
+## Decentralized
+
+Many payment processors claim to have no middleman. Saying the funds go directly to your wallet or instant settlement. However if the processor claims any of the following, they are likely operating as a **middleman**:
+
+* Waiting time for a merchant to receive payment is longer than sufficient blockchain confirmation.
+* The payment processor combines customer payments before sending to the merchant's wallet.
+* If there are any kind of limits on transaction volume for the merchant.
+* If the payment processor can decline, reject or alter a payment after being sent from a customer's wallet.
+* If the payment processor has terms and conditions stating they can hold or freeze your account.
+* Fees for using the payment processor are automatically taken out from the customer's payment to the merchant. 
+
+Payment processors perform some middleman actions by using a **custodial wallet**. A payment processor can use an internal custodial wallet for altering customer payments before routing them to merchants. This is how they can collect fees, hold payments for verification and processing, etc. This type of wallet is an intermediary between the merchant wallet and the customer wallet. It's the middleman wallet.
+
+The payment processor may also provide a custodial wallet for the merchant to use. As mentioned above, this is advised against because your private keys are not safe. If they claim to not save your private keys after giving them to you, it's likely you will not know the truth until it's too late. Centralized services may seem like an easier solution for the merchant. Unfortunately the trade-off is sacrifices in privacy, security and self-sovereignty which is normally obtained using the Bitcoin Network.
+
+That's why BTCPay was created. To help merchants remove third party dependencies and simply use the Bitcoin Network freely and securely. Merchants have a copy of the BTCPay software which runs on their own server or VPS of their choice and validates their own payments using their own node. It's a self-hosted Peer-to-Peer payment processor. The trade-off in this case is some technical understanding is required for initial setup.
+
+As the BTCPay community continues to grow, more deployment methods, use cases and tutorials are continually being added to make it easier for non-technical users. BTCPay is completely open source. Anyone can join the community to suggest or create improvements, features, guides, etc. Feedback is always welcome. 
+
+----
+
+## Fiat
+
+Currently, BTCPay is a processor **without fiat conversion** capability. As a merchant, this may be a difficult if business costs require fiat. Not providing fiat conversion allows BTCPay merchants to avoid KYC and AML identification verification. This also allows BTCPay to be free and availible for anyone to use. 
```

Comment:
However, mention that it's something on the roadmap and that since merchants are always the owners of their public keys, hence the coins, they can always freely convert their coins manually, but for now there's no instant-fiat conversion.